apps/beeai-server/src/beeai_server/infrastructure/kubernetes/default_templates/build-provider-job.yaml

```diff
@@ -68,7 +80,13 @@ spec:
             - name: image-tar
               mountPath: /tmp
           securityContext:
+            allowPrivilegeEscalation: false
```

Comment:
Both builder implementations require priviledges, `kaniko` must be run as root and buildx has several options, but none of them allow these security constraints.